graceful_controller_ros/src/graceful_controller_ros.cpp

```diff
@@ -407,9 +407,20 @@ bool GracefulControllerROS::computeVelocityCommands(geometry_msgs::Twist& cmd_ve
   double max_vel_x = max_vel_x_;
   if (!odom_helper_.getOdomTopic().empty())
   {
-    max_vel_x = robot_vel_x + (acc_lim_x_ * acc_dt_);
-    max_vel_x = std::min(max_vel_x, max_vel_x_);
-    max_vel_x = std::max(max_vel_x, min_vel_x_);
+    if (robot_vel_x > max_vel_x)
+    {
+      // If our velocity limit has recently changed,
```

Comment:
a bit pedantic, but even more accurate wording would be: "if our velocity limit has recently decreased"